docs/stdlib/array.rst

```diff
@@ -161,14 +161,17 @@ Reference
 
 .. eql:operator:: arrayslice: array<anytype> [ int64 : int64 ] -> anytype
 
-    Array slicing.
+    Slices an array of :eql:type:`anytype` containing a fixed range of
+    :eql:type:`int64` into a representable reference of the array's elements.
```

Comment:
This will be an error, mush be a single sentence/paragraph fitting in 79 characters.